src/app/interceptors/base-http.interceptor.ts

```diff
@@ -22,21 +21,22 @@ export class BaseHttpInterceptor implements HttpInterceptor {
     private localStorageManagerService: LocalStorageManagerService,
     private spinnerService: SpinnerService,
     private router: Router
-  ) { }
+  ) {}
 
   private currentRequest!: HttpRequest<any>;
 
   public intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
+    if (this.optimisticUpdateCaseUrls.includes(request.url)) {
```

Comment:
if (this.EXCLUDED.includes(request.url)) {
      return this.usualHandle(request, next);
}